Firestore/Source/Local/FSTLevelDBMigrations.mm

```diff
@@ -57,6 +58,24 @@ static void SaveVersion(FSTLevelDBSchemaVersion version, FSTWriteGroup *group) {
   [group setData:version_string forKey:key];
 }
 
+static void AddTargetCount(std::shared_ptr<DB> db, FSTWriteGroup *group) {
```

Comment:
Comments please. What's it doing and what's using the outcome? This is especially important here since we're programming by side-effect.

Reply:
Done.